util/codec/codec.go

```diff
@@ -1319,7 +1319,8 @@ func ConvertByCollationStr(str string, tp *types.FieldType) string {
 
 // HashCode encodes a Datum into a unique byte slice.
 // It is mostly the same as EncodeValue, but it doesn't contain truncation or verification logic in order
-// 	to make the encoding lossless.
+//
+//	to make the encoding lossless.
```

Comment:
I think some of these can be inlined or so? It must be wrapped by the text editor.

Reply:
I think it was gofmt's mistake to take this as the code block. So I append this word on the previous line.